inference-engine/tests/functional/plugin/gpu/shared_tests_instances/behavior/plugin/core_integration.cpp

```diff
@@ -153,6 +153,30 @@ INSTANTIATE_TEST_SUITE_P(
         ::testing::Values("GPU")
 );
 
+using IEClassGetMetricTest_GPU_MEMORY_STATISTICS = BehaviorTestsUtils::IEClassBaseTestP;
+TEST_P(IEClassGetMetricTest_GPU_MEMORY_STATISTICS, GetMetricAndPrintNoThrow) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    InferenceEngine::Core ie;
+    InferenceEngine::Parameter p;
+
+    InferenceEngine::ExecutableNetwork exeNetwork = ie.LoadNetwork(simpleCnnNetwork, deviceName);
+
+    ASSERT_NO_THROW(p = ie.GetMetric(deviceName, GPU_METRIC_KEY(MEMORY_STATISTICS)));
+    std::map<std::string, uint64_t> t = p;
```

Comment:
It'd be great to have more functional tests and some result value checks.
E.g.:
1. create core -> get stat -> check that it's 0 -> load network -> check that it's != 0 -> free network -> check that it's zero again.
2. load network -> get stat -> load second net instance -> check that mem stat doubles
etc

Reply:
@vladimir-paramuzov I have simple question regarding how to free network for increasing test coverage. Is there a way or sample to manually free exeNetwork created through LoadNetwork before it is automatically released? Furthermore, the shared pointer of exeNetwork, impl_ was found to be inaccessible.

Reply:
@vladimir-paramuzov ping again. :-)

Reply:
Option 1:
```cpp
    // check memory statistics
    {
        auto exec_net1 = ie.LoadNetwork(net1, CommonTestUtils::DEVICE_GPU);
        {
             auto exec_net2 = ie.LoadNetwork(net2, CommonTestUtils::DEVICE_GPU);
            // check memory statistics
        }
        // check memory statistics
    }
    // check memory statistics
```
Option 2:
```cpp
    // check memory statistics
    std::shared_ptr<ExecutableNetwork> exec_net1 = nullptr;

    {
        auto exec_net_tmp = ie.LoadNetwork(net, CommonTestUtils::DEVICE_GPU);
        exec_net1.reset(new ExecutableNetwork(exec_net_tmp));
    }
    // check memory statistics
    exec_net1.reset(nullptr);
    // check memory statistics 
```

Reply:
Added test cases using the first option.